SDL2/SDL2-2.26.1.json

```diff
@@ -0,0 +1,17 @@
+{
+  "../libdecor/libdecor-0.1.1.json",
+  
+  "name": "SDL2",
+  "buildsystem": "autotools",
+  "config-opts": ["disable-static"],
```

Comment:
I think you meant `--disable-static` and then you won't need to cleanup `libSDL2main.a` because it won't exist.

It would be nice for you to test this manifest.